### PR TITLE
Eng 1999 Sort Envelopes

### PR DESCRIPTION
**Pull request by PaulDHenson**
Added a sort to the envelopes when they are being fetched from the DB. The envelopes are sorted by their group number which will keep them together, but their match stamp numbers will still be random.

### CODE REVIEW

File: react/game_area.js [hunk 1]
```diff
@@ -121,6 +121,8 @@ class GameArea extends Component {
   async updateGame() {
     const response = await fetch(`/api/game-state/${this.gameId}`)
     const json = await response.json();
+    json.envelopes.sort((a, b) => (a.groupNumber > b.groupNumber) ? 1 : -1);
+    console.log("Line 128 ");
```
Comment (BFisch14) [inline react/game_area.js:125]: maybe get rid of frontend comments

**Review by BFisch14 — COMMENTED** [review]
Looks good, not sure on frontend comments

File: react/game_area.js (hunk 1 above)
Comment (PaulDHenson) [reply react/game_area.js]: Just cleaned that up, thanks!

**Review by BFisch14 — APPROVED** [review]
Looks better!

**Pull request merged**